libraries/vendor/joomla/image/src/Image.php

```diff
@@ -211,10 +211,13 @@ public static function getImageFileProperties($path)
 			throw new \InvalidArgumentException('The image file does not exist.');
 		}
 
-		// Get the image file information.
-		$info = getimagesize($path);
+		if (!is_dir($path))
```

Comment:
Either way you need to make the pr with the changes in the vendor directory to the upstream package or catch that case in the media manager. Changes here will be overwritten on the next composer update.